### PR TITLE
Add posibility to override/extend html class for the containing div

### PR DESCRIPTION
**Pull request by Jakub-Fajkus**
It would be nice to be able to add a custom html class to the containing div of the whole grid. In our use case, we would use that to add a html class to properly display a spinner when an user clicks the `Filter` button or uses the drag&drop sorting of the rows.

### CODE REVIEW

**Comment by f3l1x** [comment]
👍wdyt @paveljanda?

File: src/templates/datagrid.latte [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * @param InlineEdit|null $inlineAdd   Inline add data
  *}
 
-<div class="datagrid datagrid-{$control->getFullName()}" data-refresh-state="{link refreshState!}">
+<div class="{block grid-class}datagrid datagrid-{$control->getFullName()}{/block}" data-refresh-state="{link refreshState!}">
```
Comment (paveljanda) [inline src/templates/datagrid.latte]: Maybe pls `{block datagrid-class}`? (There probably more references to a "datagrid" than to a "grid"). Thanks

Comment (Jakub-Fajkus) [reply src/templates/datagrid.latte]: Sure, everything is called datagrid... changed that

**Comment by paveljanda** [comment]
Nice thanks

**Comment by paveljanda** [comment]
@Jakub-Fajkus Can you please update also the docs? Add somewhere that this feature is available. Thx

**Comment by Jakub-Fajkus** [comment]
@paveljanda Added a new section `Containing div class definition` (not sure about the name, though). I also added the missing class to section `Table class definition`.

**Comment by paveljanda** [comment]
Thanks!

**Pull request merged**